### DISCUSSION TITLE
Missing several classes

### DISCUSSION THREAD

**Discussion by Jarrodsz** [rails/tailwindcss-rails#204] [Q&A]
Implemented like readme.md

But the problem is im missing all kind of classes like border-red-500 does not exists in the build tailwind.css 

How to resolve this?

I even tried creating a new rails 7 application with the prefix for tailwind css and it gives the same result. No border-red classes only border-grey exists :S

```
<div class ="border-2 border-red-500">
  xxx
  x
  x
  x
  x
</div>
```

**Comment by flavorjones** [comment]
Hi, it's important to remember that tailwind is configured in `config/tailwind.config.js`. In rails, by default it will be configured to look for TailwindCSS classes in these files:

```
    './public/*.html',
    './app/helpers/**/*.rb',
    './app/javascript/**/*.js',
    './app/views/**/*.{erb,haml,html,slim}'
```

You'll need to _use_ a tailwind class in one of these files, and then re-generate your CSS using `rake tailwindcss:build` or `rake tailwindcss:watch`, in order for the class to appear in `app/assets/builds/tailwind.css`.

Please see https://tailwindcss.com/docs/content-configuration for more information on how this works.

Does this response make sense?

**Reply by Jarrodsz** [reply]
Thank you @flavorjones 

I have followed the instructions on the website of tailwindui, since I could not get it work I created a new rails app.
I also tried the new app and added some styles. Default rails tailwind runs with ./bin/dev and should compile the assets. 

Also made sure all the paths are inside the tailwind config like :

''' './app/views/**/*.{erb,haml,html,slim}' '''

It still does not seem to compile all my used classes. I have a layout that works when I include the Tailwind CDN but not when Im building the tailwind with this gem solution. :( really lost on this. 

Any pointers as how I could debug?

**Reply by flavorjones** [reply]
Can you help me reproduce what you're seeing? I'm able to create a new rails app and this works for me, so I need to understand what we are doing differently.
